src/array/cpu/spmm.h

```diff
@@ -43,7 +43,39 @@ using AccType = typename std::conditional<
  *       for the computation of different nodes.
  */
 template <typename IdType, typename DType, typename Op>
-void SpMMSumCsrNaive(
+typename std::enable_if<!std::is_same<DType, BFloat16>::value, void>::type
+SpMMSumCsrNaive(
```

Comment:
could you add notes for the main difference of bf16 and non-bf16 implementation?